index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+require_once ("Autor.php");
+require_once ("ImprimirAutir.php");
```

Comment:
```suggestion
require_once ("ImprimirAutor.php");
```
Cambiar "Autir" por "Autor"

Reply:
corregido